package.json

```diff
@@ -39,7 +40,8 @@
     "jquery": "2.2.1",
     "lodash": "4.15.0",
     "n3": "0.6.1",
-    "webcola": "3.1.3"
+    "webcola": "3.1.3",
+    "whatwg-fetch": "^2.0.2"
```

Comment:
jquery has no dependency on whatwg-fetch. I had to do additional calls and would prefer that no new code depends on jquery. I removed $.ajax calls from sparql provider.